tagging/models.py

```diff
@@ -474,14 +474,17 @@ class Meta:
     def __unicode__(self):
         return self.name
 
+    def __str__(self):
+        return self.__unicode__()
```

Comment:
why do we need `__str__` too? is this for py3?

Reply:
Yes, python3.